scout/parse/omim.py

```diff
@@ -38,6 +38,67 @@ def parse_omim_line(line, header):
     return omim_info
 
 
+def parse_genemap2_phenotypes(phenotype_entry, mim_number=None):
+    """Parse the Phenotype entry of a genemap2 line
+
+    Returns a list with the relevant phenotypes.
+    The phenotype entries are separated by ';'
+
+    If a special symbol is used in the description it indicates the disease status.
+
+    """
+    parsed_phenotypes = []
+
+    for phenotype_info in phenotype_entry.split(";"):
+        if not phenotype_info:
+            continue
+        phenotype_info = phenotype_info.lstrip()
```

Comment:
Why not a simple strip?

Reply:
Guess I only want to strip on the left side but it might not matter